README.md

```diff
@@ -326,6 +327,19 @@ you can check the
 [vc-test-suite-implementations](https://github.com/w3c/vc-test-suite-implementations)
 library.
 
+### Allure Reporting
+It's also possible to generate local allure reports for analyzing and debuging results.
+
+To run the tests and browse the report, use the following commands:
+```bash
+# Running the tests
+npx mocha tests/
+
+# Running the reporting server
+allure serve allure-results
+
```

Comment:
```suggestion
```

Reply:
Removed the newline within the bash codeblock